### PR TITLE
[gce optimization] rename some patch fields

### PR DESCRIPTION
**Pull request by Qi77Qi**
\<your comments for this PR go here\>

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -1991,21 +1991,21 @@ components:
...
           properties:
-            updatedMasterMachineType:
+            masterMachineType:
               type: string
               description: >
                 Optional, the machine type determines the number of CPUs and memory
                 for the cluster master node. For example "n1-standard-16" or "n1-highmem-64".
                 To decide which is right for you, see https://cloud.google.com/compute/docs/machine-types
-            updatedDiskSize:
+            diskSize:
```
Comment (zarsky-broad) [inline http/src/main/resources/swagger/api-docs.yaml]: `masterDiskSize`?

**Review by rtitle — APPROVED** [review]
Thanks - I had decided to name them `updated*` but I see the logic in keeping the same names.

**Comment by Qi77Qi** [comment]
fiab start failure...jenkins retest

**Comment by rtitle** [comment]
@Qi77Qi i think it's a known issue affecting all fiab-start runs… Rob M is working on a fix.

**Comment by Qi77Qi** [comment]
jenkins retest

**Pull request merged**